graphql-database-manager-test/src/test/java/com/fleetpin/graphql/database/manager/test/DynamoDbHistoryTest.java

```diff
@@ -433,6 +434,23 @@ void testStartWithToUpdatedAtQuery(final Database db, final HistoryProcessor his
 		Assertions.assertEquals(revision4Time, history.get(3).getUpdatedAt());
 	}
 	
+	@TestDatabase
+	void testNoHistoryTable(final Database db, final HistoryProcessor historyProcessor) throws InterruptedException, ExecutionException {
+
+		var table1 = new NoHistorySameNameTable("revision1");
+		table1.setId("testTable1");
+        db.put(table1).get();
+        table1 = new NoHistorySameNameTable("revision2");
+        table1.setId("testTable1");
+        table1.setRevision(1);
+        db.put(table1).get();
+        
+        historyProcessor.process();
+		var history = db.queryHistory(QueryHistoryBuilder.create(SameNameTable.class).id("testTable1").build()).get();
+		Assertions.assertEquals(0, history.size());
+
+	}
```

Comment:
looks good would just add some tests testing the failure cases and confirming they through errors.
I.e. setting both startWith and id
trying to do a history query on an object that does not have the `@History` annotation
then good to go 👍